mne/realtime/tests/test_mockclient.py

```diff
@@ -29,16 +31,44 @@ def test_mockclient():
     data = epochs.get_data()
 
     rt_client = MockRtClient(raw)
+    # choose "large" value, should always be longer than execution time of
+    # get_data()
+    isi_max = 5
     rt_epochs = RtEpochs(rt_client, event_id, tmin, tmax, picks=picks,
-                         isi_max=0.5)
+                         isi_max=isi_max)
 
     rt_epochs.start()
     rt_client.send_data(rt_epochs, picks, tmin=0, tmax=10, buffer_size=1000)
 
+    # get_data() should return immediately and not wait for the timeout
+    start_time = time.time()
     rt_data = rt_epochs.get_data()
-
+    retrieval_time = time.time() - start_time
+    assert_true(retrieval_time < isi_max)
     assert_true(rt_data.shape == data.shape)
     assert_array_equal(rt_data, data)
+    assert_true(len(rt_epochs) == len(epochs))
```

Comment:
you can't tell from this file but we have stopped using `nose`-based asserts like `assert_true` and `assert_equal` now that we use `pytest`, instead just do:
```
assert len(rt_epochs) == len(epochs)
```

Reply:
Ok, I'll change all the tests I touched - or should I convert the whole file?
Also, do you prefer ``mne.utils._TempDir`` or the ``pytest`` ``tmpdir`` fixture?

Reply:
We have been migrating one file at a time as people are motivated to do it, so yes please feel free!

We haven't transitioned to using `tmpdir` anywhere yet, but probably should -- feel free to make that change as well and we'll probably use it as a template elsewhere.